### PR TITLE
feat(lattices): Expose PairBimorphism public.

### PR DESCRIPTION
**Pull request by rohitkulshreshtha**
Address https://github.com/hydro-project/hydroflow/issues/1229.

### CODE REVIEW

**Review by MingweiSamuel — APPROVED** [review]
`feat(lattices): ...`

**Comment by rohitkulshreshtha** [comment]
> `feat(lattices): ...`

Yeah. Changing the title doesn't work, and I can't change the commit message because I made this PR directly from GitHub. Will have to squash and merge after approval.

**Pull request merged**
